models/DataObject/Listing.php

```diff
@@ -228,7 +228,7 @@ public function key()
     }
 
     /**
-     * @return mixed|void
+     * @return mixed|null
```

Comment:
I think `@return mixed` should be enough. `null` is impossible. The `next` function will return `false` if the array is at the end.